docker-compose.yml

```diff
@@ -6,7 +6,7 @@
 # Instructions for necessary settings during setup when using docker-compose deployment can be found here:
 # TBD
 
-version: '3'
+#version: '3'
```

Comment:
@acuanico-tr-galt These doesn't seem related to chrome-php updates. Is this for your local only?